java/trace/src/main/scala/org/typelevel/otel4s/java/trace/TracerImpl.scala

```diff
@@ -33,6 +34,14 @@ private[java] class TracerImpl[F[_]: Sync](
   val meta: Tracer.Meta[F] =
     Tracer.Meta.enabled
 
+  def currentSpan: F[Span[F]] =
+    scope.current.map {
+      case TraceScope.Scope.Span(_, jSpan, ctx) =>
```

Comment:
Should we check that the context is also valid `if ctx.isValid`?

Reply:
Yes, I think so.